ui/raidboss/emulator/data/RaidEmulator.js

```diff
@@ -7,11 +7,10 @@ export default class RaidEmulator extends EventBus {
     this.options = options;
     this.encounters = [];
     this.currentEncounter = null;
-    this.playing = false;
-    this.currentTimestamp = null;
+    this.playingInterval = null;
     this.currentLogLineIndex = null;
     this.lastLogTimestamp = null;
-    this.internalTimestampTracker = null;
+    this.emulatedTimestamp = null;
```

Comment:
Is it possible to use different words to differentiate what time spaces these variables are operating in?  `emulatedTimestamp` here seems to be tracking "real time", but `currentTimestamp` and `lastLogTimestamp` seem to be "log time".

Proposing:
* `emulatedTimestamp` => `lastTickTime`
* `currentTimestamp` => `currentLogTime`
* `lastLogTimestamp` => `lastLogTime`

Reply:
`currentTimestamp` is tracking elapsed time since the start of the fight according to playback and will almost never line up with an actual log timestamp. It might be better described as `currentFightTime`.

The other two look fine.

Reply:
What about `currentTimestamp` => `elapsedTime`

Reply:
To be clear, `currentTimestamp`:

- Is a unix timestamp
- Always occurs between the first log line of the encounter and the last log line of the encounter
- Does not necessarily correspond to a given log line
- Increases every tick of the emulator
- May decrease if seeking to a previous point in the encounter

Reply:
> currentTimestamp is tracking elapsed time

:thinking:  I guess I think of "elapsed" as meaning a time delta, subtracting "current time - start time".

I agree that it doesn't correspond to a given log line, but it is still "log time".  I think of there being different time spaces that we are converting between.  There's real time (Date.now()), there's timeline time (corresponding to `fightNow` in timeline), there's log time (in the emulator, corresponding to the timestamps on logs) and there's synthetic time for lack of a better word (which is `this.timebase` and can be either real time or log time, depending on whether it's in the emulator or not).

The space in which `currentTimestamp` moves around is in "log time", in that if X < `currentTimestamp` < Y, then log X happens before log Y.

That's why I'm trying to differentiate "log time" vs "timeline time" vs "real time".

Reply:
My worry is that `currentLogTime` implies that this might correspond to a given log line due to `lastLogTime` corresponding to a given log line, but that's not the case.

Maybe instead:

- `emulatedTimestamp` => `lastTickTime`
- `currentTimestamp` => `currentLogTime`
- `lastLogTimestamp` => `lastLogLineTime`

Just to differentiate the other timestamp?

Reply:
Sounds good! :+1:

Reply:
(What happened to `lastLogLineTime`?)

Reply:
(It turns out, if you forget to stage your final commit changes, they don't actually get added to the commit. Whoops.)